global-cfo.php

```diff
@@ -18,7 +18,7 @@
  * Plugin Name:       Global CFO
  * Plugin URI:        https://github.com/TotalOnion/cloudflareonion
  * Description:       Cloudflare cache handling plugin
- * Version:           0.0.2
+ * Version:           0.0.4
```

Comment:
This versionning should probably change when the repo gets a new tag. I think.

Reply:
The issue is there's repo/satis versioning, and WP/assets versioning. So we have to change it ahead of merging PR no ?